.github/workflows/ci-uv.yml

```
@@ -0,0 +1,44 @@
+    - name: Run linting
+      run: |
+        uv run ruff check src tests
+        uv run black --check src tests
+        uv run mypy src
+
+    - name: Run tests
+      run: |
+        uv run pytest --cov=src/toady --cov-report=xml
+
+    - name: Upload coverage
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)